config/spec/mainnet.go

```diff
@@ -71,6 +72,19 @@ const (
 	// The deposit contract address on mainnet at genesis is the same as the
 	// default deposit contract address.
 	MainnetDepositContractAddress = defaultDepositContractAddress
+
+	// MainnetDeneb1ForkEpoch is the epoch at which the Deneb1 fork occurs.
+	//
+	// TODO: set to the correct epoch.
+	MainnetDeneb1ForkEpoch = 5000
```

Comment:
@calbera I believe we should do something like that for whatever testnet our e2e tests use. So that we check this fork in e2e tests as well

Reply:
Right now its devnet, but we will have to use a shorter number in devnet so that e2e doesn't spin until a really large block number